### PR TITLE
Added support for peer review flag in GUI, testing, and database

### PR DESCRIPTION
**Pull request by WChrisK**

### CODE REVIEW

**Comment by WChrisK** [comment]
Test failures will be fixed tomorrow (not ready for integration yet)

File: db/migrate/20160518004516_add_peer_review_to_assignments.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddPeerReviewToAssignments < ActiveRecord::Migration
+  def change
+    add_column :assignments, :peer_review, :boolean, default: false
```
Comment (david-yz-liu) [inline db/migrate/20160518004516_add_peer_review_to_assignments.rb]: Again, I don't want to re-use the name "peer_review" too much. Maybe change the column name to "has_peer_review". Also, put in a null: false.

**Comment by WChrisK** [comment]
I assume it's implied that I did this, but to confirm: It does save to the database (and checking it w/ selecting assignments from psql) and saves/loads properly.

**Comment by david-yz-liu** [comment]
@WChrisK please add translations to the other files (you can just use Google Translate here).

**Pull request merged**